src/document.ts

```diff
@@ -35,14 +35,16 @@ import * as yaml from 'js-yaml';
 import type { Document } from './schema';
 import * as schema from './schema.json';
 
-// build a swagger validator from the official v2.0 schema
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const schemaValidator = jsonValidator(schema as any);
+let schemaValidator: ReturnType<typeof jsonValidator> | undefined;
```

Comment:
this file contains the only real change.  everything else is eslint/prettier fixes

Reply:
lgtm